main.go

```diff
@@ -206,6 +238,20 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err = (&infrav1exp.OCIManagedCluster{}).SetupWebhookWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create webhook", "webhook", "OCIManagedCluster")
+		os.Exit(1)
+	}
+
+	if err = (&infrav1exp.OCIManagedControlPlane{}).SetupWebhookWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create webhook", "webhook", "OCIManagedControlPlane")
+		os.Exit(1)
+	}
+
+	if err = (&infrav1exp.OCIManagedMachinePool{}).SetupWebhookWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create webhook", "webhook", "OCIManagedMachinePool")
+		os.Exit(1)
+	}
```

Comment:
yes, users can still create those objects as the crds are aded, so if we dont add the webhook, we will have invalid objects, and then if user restarts the pod with the flag set to true, it will lead to error.
We are only disabling the reconciliation right.